api/events/schema.py

```diff
@@ -200,3 +202,24 @@ class Mutation(graphene.ObjectType):
             [required]\n- start_time: The start time of the calendar event\
             [required]\n- end_time: The field with the end time of the calendar\
              event[required]")
+
+
+class Query(graphene.ObjectType):
+    all_events = graphene.List(
+        Events,
+        start_date=graphene.String(),
+        end_date=graphene.String(),
+        description="Query that returns a list of all events")
+
+    @Auth.user_roles('Admin', 'Default User')
+    def resolve_all_events(self, info, **kwargs):
+        start_date, end_date = CommonAnalytics.all_analytics_date_validation(
+            self, kwargs['start_date'], kwargs['end_date']
+        )
+        query = Events.get_query(info)
+        all_events = query.filter(
```

Comment:
What happens when I want to query only today's events?

Reply:
At the moment 
<img width="1191" alt="Screenshot 2019-05-29 at 00 34 39" src="https://user-images.githubusercontent.com/33450849/58514062-958cce80-81a9-11e9-8b99-0eec49740618.png">

My sync all events does not work at the moment though